<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCResistance</class>
 <widget class="QWidget" name="DCResistance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zhizu</string>
  </property>
  <widget class="QTextEdit" name="textEdit1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;SR3320直流电阻测试仪&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbParameter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电流选择</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>58</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="cobOutputCurrent">
      <item>
       <property name="text">
        <string>10A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100mA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5mA</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="grbData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>351</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>数据显示</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string notr="true">设备编号(名称)：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>选择绕组：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>选择分接：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A相阻值：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B相阻值：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C相阻值：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>序号：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>测试日期：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>测试时间：</string>
     </property>
     <property name="font">
      <font>
       <family>Adobe Caslon Pro</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lneDeviceID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cobWinding">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbShunting">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneRValueA">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneRValueB">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneRValueC">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneSerialNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneMeasuringDate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneMeasuringTime">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>数据存储</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打印报表</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnHandshaking">
        <property name="text">
         <string>联机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTesting">
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrintReport">
        <property name="text">
         <string>打印</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnCurrentSetting">
        <property name="text">
         <string>电流设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeviceReset">
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="plainText">
    <string>	正在切换信道</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>电  流：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>阻  值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneCurrentShow">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneRValueShow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="dcrStatusBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid lightGray</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>textEdit1</zorder>
  <zorder>grbParameter</zorder>
  <zorder>grbData</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>lneRValueShow</zorder>
  <zorder>lneCurrentShow</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>dcrStatusBar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
